config/releases.exs

```diff
@@ -99,6 +99,8 @@ config :meadow,
   },
   environment: :prod,
   digital_collections_url: get_required_var.("DIGITAL_COLLECTIONS_URL"),
+  iiif_cloudfront_distribution_id: get_required_var.("IIIF_CLOUDFRONT_DISTRIBUTION_ID"),
+  iiif_cloudfront_version: get_required_var.("IIIF_CLOUDFRONT_VERSION"),
```

Comment:
I think a module attribute would be better for this than a variable, if you don't want to hardcode it. These AWS API version strings don't change very often at all, and there's no penalty for using an old one as long as your data conforms to the shape of that version. Which it will unless we change it, at which point we would presumably change the hardcoded version string as well.

As another example, we have `"Version": "2012-10-17"` hardcoded in our IAM policy documents all over our Terraform code.

Reply:
@mbklein - Updated, PTAL. Terraform reapplied on staging

Reply:
`releases.exs:103` still needs to go away or the container won't start due to the missing `IIIF_CLOUDFRONT_VERSION` variable.